osquery/core/darwin/iokit.hpp

```diff
@@ -0,0 +1,54 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#pragma once
+
+#include <atomic>
+#include <iomanip>
+
+#include <CoreServices/CoreServices.h>
+#include <IOKit/IOKitLib.h>
+
+#include "osquery/core/conversions.h"
```

Comment:
Almost, flip this with the SDK header include and put a space between the two and you'll be set :)